examples/are-you-alive/Makefile

```diff
@@ -0,0 +1,15 @@
+# We are using the "dev" project in our registry for this
+NAME   := "us.gcr.io/planetscale-dev/are-you-alive"
```

Comment:
I think this should be `us.gcr.io/planetscale-vitess/are-you-alive`.  Does that sound good to you @dctrwatson?

Reply:
If this is meant for public consumption, yes :)

Reply:
Ok, updated, but now I need to fix the build.